setup.py

```diff
@@ -2,9 +2,13 @@
     import regex as re
 except ImportError:
     import re
-from setuptools import find_packages, setup
+from setuptools import setup  # , find_packages  # unused
 
-__version__ ,= re.findall('__version__ = "(.*)"', open('lark/__init__.py').read())
```

Comment:
The reason I'm using `re` is so setup.py could be run without having to install dependencies.

See pt 6. https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
But `import lark` is a relative import.

Reply:
I see.

Reply:
No? A relativ import starts with a `.`, so `import .lark` would be relative. And why does that matter: You are still importing the entire package, including all submodules that might import dependencies.

Reply:
Oh ok.